app/Http/Controllers/IndexController.php

```diff
@@ -71,6 +79,7 @@ public function phpinfo()
 	 */
 	public function gallery()
 	{
+		$this->symLinkFunctions->remove_outdated();
```

Comment:
So will this run every time a new user connects to the gallery? Perhaps there should be some throttling? Then again, it's a single DB query...

Reply:
well it is a bit more than a single DB query as it's a query for every picture symlinked...